config/macros.tex

```diff
@@ -153,10 +153,6 @@
 \newcommand{\mimcEnc}{\ensuremath{\algostyle{\mimc{}\pcmathhyphen Encrypt}}\xspace} % Mimc block cipher encrypt
 \newcommand{\mimcMP}{\ensuremath{\algostyle{\mimc{}\pcmathhyphen MP}}\xspace} % Mimc Miyaguchi-Preneel compression function
 \DeclareRobustCommand{\mimcMPPrime}[1]{\ensuremath{\mimcMP\ifthenelse{\equal{#1}{}}{}{_{#1}}}\xspace} % Mimc Miyaguchi-Preneel compression function mod r
-\newcommand{\mimcSeven}{\ensuremath{\algostyle{\mimc[7]}}\xspace} % Mimc block cipher with (e=7)
```

Comment:
Instead of macros for specific instantiations, the general form is used to define `\mkhash` and the relevant constants for each curve of interest are given.